app/src/main/res/layout/main_activity.xml

```diff
@@ -6,7 +6,9 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:fitsSystemWindows="true"
-    tools:context="protect.card_locker.MainActivity">
+    tools:context="protect.card_locker.MainActivity"
+    android:focusable="true"
+    android:focusableInTouchMode="true">
```

Comment:
Can you explain what this change is for a little?

Reply:
I honestly don't know why this works and setting default focus on the edittext element to false doesn't, but I found this on https://android--code.blogspot.com/2015/08/android-edittext-auto-focus.html?m=1 and in my testing it worked reliably. I will add a comment to explain.